InputDate/InputDateWeekday/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Text from '../../Text';
+
+const WeekDay = ({
+  weekdayLenght,
+  weekday,
+  className,
+  localeUtils,
```

Comment:
I think it would be better to import this directly rather than pass it down as props?

```
import DayPicker from 'react-day-picker';
// or if the library allows 
import { LocaleUtils } from 'react-day-picker';
```

Reply:
It is one of the 4 props the `weekdayElement` receives automatically from the plugin